utils/fake_data_generators.py

```diff
@@ -0,0 +1,11 @@
+from faker import Faker
+
+
+def get_fake_title():
+    fake_name = Faker().first_name()
+    return fake_name
```

Comment:
Would be better to name this method getFakeFirstName not title
Because you can reuse it not only for generating title.